### PR TITLE
Add necessary file to enable dependabot in Settings

### PR DESCRIPTION
**Pull request by garettarrowood**
To enable Dependabot, this yml file must first exist. This file is not yet configured. @jasonodoom has interest in setting this up properly.

### CODE REVIEW

**Comment by garettarrowood** [comment]
This does not resolve #7 , but just unblocks it.

**Comment by jasonodoom** [comment]
@cflipse can you please provide input so I will know how to proceed?

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "" # See documentation for possible values
```
Comment (cflipse) [inline .github/dependabot.yml:8]: this should specify as `bundler`

**Comment by cflipse** [comment]
Sure, go for it.  

I don't _think_ we're using anything other than the bundler ecosystem, though _maybe_ the docker ecosystem is worth adding?  I'm not terribly familiar with what this will do with just `docker-compose` files though, so that might be pointless.

File: .github/dependabot.yml (hunk 1 above)
Comment (jasonodoom) [reply .github/dependabot.yml:8]: glad I waited for your feedback :)

**Pull request merged**